prime-angular-frontend/src/app/config/config.service.ts

```diff
@@ -127,6 +127,16 @@ export class ConfigService implements IConfigService {
     return of({ ...this.configuration });
   }
 
+  public getPrescriberIdType(licenceCode: number): PrescriberIdTypeEnum {
+    const prescriberIdTypes = this.licenses
+      .filter(licenseConfig => licenseConfig.code === licenceCode)
+      .map(licenseConfig => licenseConfig.prescriberIdType);
+
+    return (prescriberIdTypes.length)
+      ? prescriberIdTypes[0]
+      : PrescriberIdTypeEnum.NA;
+  }
+
```

Comment:
compared to other methods in the file, this method seems more like belonging to a mapping or pipe rather than returning config.

Reply:
You're right, was thinking to put method into some utility class but `ConfigService` also has the necessary information, i.e. the `LicenseConfig` information

Reply:
The ConfigService is supposed to be a "low-level" provider for Lookups.  It shouldn't have utility methods in it like @johnnyl6  mentions.  The `EnrolmentService` would be a good place to have this validation alongside canRequestRemoteAccess, hasAllowedRemoteAccessCareSetting, hasAllowedRemoteAccessLicences, and shouldShowCollegePrefix.